### ISSUE TITLE
GitHub Registration Issue Resolved

### ISSUE THREAD

**Issue by Humdinger2019** [matdombrock/SA-Testing#15]
I am registered and verified, and am doing this on my phone as the ExpressLaw is not up yet.
Am I posting this "test issue" in the correct spot?

**Comment by matdombrock** [comment]
Yep this all looks good to me. Its not ideal that you have to use it from your phone but at least it's working.

**Comment by Humdinger2019** [comment]
Yes, I will use the laptop again once ExpressLaw is up. But, want to move forward on this after being stuck in the weeds with silly technical issues for a time.
GitHub seems pretty straightforward. So, as I test the software, I make a list of issues, the place them separately under issues in GitHub for you to see, diagnose. And, when needed, discuss in more detail with me.

Sent from my Verizon Motorola Smartphone
On Sep 17, 2019 10:10 AM, Mathieu Dombrock <notifications@github.com> wrote:

Yep this all looks good to me. Its not ideal that you have to use it from your phone but at least it's working.

**Comment by matdombrock** [comment]
That's exactly right. You got it.

**Comment by Humdinger2019** [comment]
Good. Let me next steps...

Sent from my Verizon Motorola Smartphone
On Sep 17, 2019 10:58 AM, Mathieu Dombrock <notifications@github.com> wrote:

That's exactly right. You got it.

**Comment by matdombrock** [comment]
Are you asking what the next steps should be?

**Comment by Humdinger2019** [comment]
More, just letting you know that I am available for the next steps, when you are ready...

Sent from my Verizon Motorola Smartphone
On Sep 17, 2019 12:02 PM, Mathieu Dombrock <notifications@github.com> wrote:

Are you asking what the next steps should be?

**Comment by matdombrock** [comment]
Ok I think that after you've checked out the [Read Me](https://github.com/matdombrock/SA-Testing/blob/master/README.md) you should try logging into the site and just make sure that you can  get into some mediator and client accounts.
